src/Native/Runtime/GCHelpers.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "common.h"
 #include "gcenv.h"
+#include "../gc/env/gcenv.ee.h"
```

Comment:
Other places have this include under `FEATURE_STANDALONE_GC`. Remove the `FEATURE_STANDALONE_GC` from them to match this? (CoreRT does not have the support for standalone GC.)

Also, you may delete `src\Native\Runtime\gcenv.ee.h` - it does not seems to be used.